Week3/homework/step2-6.js

```diff
@@ -6,11 +6,13 @@ const arr3d = [[[1, 2], [3, 4]], [[5, 6], [7, 8]]];
 function flattenArray2d(arr) {
   // Replace this comment and the next line with your code
   console.log(arr);
+  return arr.flat([2]);
```

Comment:
This `.flat` array method is rather new and not universally supported by all browsers. Can you revise this code without using `.flat`?

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/flat#Browser_compatibility